cfg/wxwidgets.cfg

```diff
@@ -4483,6 +4483,7 @@
       <function name="replace" action="change"/>
       <function name="Replace" action="change"/>
       <function name="reserve" action="change-internal"/>
+      <function name="shrink_to_fit" action="change-internal"/>
```

Comment:
Just recently added to wxWidgets with commit https://github.com/wxWidgets/wxWidgets/commit/ea965327ebcf11e8c507f62e75a18742b3db2869